src/OrchardCore.Modules/OrchardCore.Redis/Services/RedisService.cs

```diff
@@ -10,21 +10,23 @@ namespace OrchardCore.Redis.Services
 {
     public class RedisService : ModularTenantEvents, IRedisService, IDisposable
     {
-        private readonly IOptions<RedisOptions> _options;
+        private readonly RedisOptions _options;
         private readonly ILogger _logger;
 
         private readonly SemaphoreSlim _semaphore = new SemaphoreSlim(1);
 
         public RedisService(IOptions<RedisOptions> options, ILogger<RedisService> logger)
         {
-            _options = options;
+            _options = options.Value;
             _logger = logger;
 
             InstancePrefix = options.Value.InstancePrefix;
         }
 
         public IConnectionMultiplexer Connection { get; private set; }
-        public string InstancePrefix { get; private set; }
+
+        public string InstancePrefix { init; get; }
```

Comment:
Why do you need this change?

Reply:
`InstancePrefix` is only readonly and set in the constructor, right?

Reply:
yes

Reply:
So you can remove the init only setter as we will never use an object initializer on `RedisService`, `new RedisService(...) { InstancePrefix = ... }`, so I suggest.

    public string InstancePrefix { get; }